### PR TITLE
[Examples] Update react-md examples for v2

### PR DESCRIPTION
**Pull request by mlaursen**
Hi, this is the new PR replacing #15590 that updates the `with-react-md` example to use react-md@v2 as well as adding a new `with-react-md-typescript` example. I copied the `tsconfig.json` from the `with-typescript` example and tried to follow the naming conventions/standards found in other examples, but let me know if I should update anything to better match the naming conventions/standards for this repo.

### CODE REVIEW

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| buildDuration | 13.4s | 13.3s | -69ms |
| nodeModulesSize | 65.5 MB | 65.5 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.511 | 2.568 | ⚠️ +0.06 |
| / avg req/sec | 995.7 | 973.64 | ⚠️ -22.06 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.46 | 1.556 | ⚠️ +0.1 |
| /error-in-render avg req/sec | 1712.35 | 1607.03 | ⚠️ -105.32 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| 677f882d2ed8..b7a9.js gzip | 10.2 kB | 10.2 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-935e413..31c4.js gzip | 6.76 kB | 6.76 kB | ✓ |
| webpack-488d..c0e7.js gzip | 751 B | 751 B | ✓ |
| Overall change | 56.8 kB | 56.8 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 6.09 kB | 6.09 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-4b8f4a8..dule.js gzip | 5.84 kB | 5.84 kB | ✓ |
| webpack-4f62..dule.js gzip | 751 B | 751 B | ✓ |
| Overall change | 51.8 kB | 51.8 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| polyfills-05..1236.js gzip | 30.8 kB | 30.8 kB | ✓ |
| Overall change | 30.8 kB | 30.8 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| _app-8f5f611..1f7b.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-a98d9..5cb7.js gzip | 3.45 kB | 3.45 kB | ✓ |
| hooks-f7f3d0..7465.js gzip | 887 B | 887 B | ✓ |
| index-08fb3f..c0e9.js gzip | 227 B | 227 B | ✓ |
| link-6f8445b..99e1.js gzip | 1.3 kB | 1.3 kB | ✓ |
| routerDirect..8aa1.js gzip | 284 B | 284 B | ✓ |
| withRouter-f..e777.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.72 kB | 7.72 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| _app-669dbe5..dule.js gzip | 626 B | 626 B | ✓ |
| _error-d5979..dule.js gzip | 2.3 kB | 2.3 kB | ✓ |
| hooks-805c40..dule.js gzip | 387 B | 387 B | ✓ |
| index-6ba5a4..dule.js gzip | 226 B | 226 B | ✓ |
| link-91516ae..dule.js gzip | 1.25 kB | 1.25 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-d..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.36 kB | 5.36 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| _buildManifest.js gzip | 274 B | 274 B | ✓ |
| _buildManife..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 556 B | 556 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| index.html gzip | 945 B | 945 B | ✓ |
| link.html gzip | 953 B | 953 B | ✓ |
| withRouter.html gzip | 940 B | 940 B | ✓ |
| Overall change | 2.84 kB | 2.84 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| buildDuration | 14.8s | 14.8s | -23ms |
| nodeModulesSize | 65.5 MB | 65.5 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| 677f882d2ed8..b7a9.js gzip | 10.2 kB | 10.2 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-935e413..31c4.js gzip | 6.76 kB | 6.76 kB | ✓ |
| webpack-488d..c0e7.js gzip | 751 B | 751 B | ✓ |
| Overall change | 56.8 kB | 56.8 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 6.09 kB | 6.09 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-4b8f4a8..dule.js gzip | 5.84 kB | 5.84 kB | ✓ |
| webpack-4f62..dule.js gzip | 751 B | 751 B | ✓ |
| Overall change | 51.8 kB | 51.8 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| polyfills-05..1236.js gzip | 30.8 kB | 30.8 kB | ✓ |
| Overall change | 30.8 kB | 30.8 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| _app-8f5f611..1f7b.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-a98d9..5cb7.js gzip | 3.45 kB | 3.45 kB | ✓ |
| hooks-f7f3d0..7465.js gzip | 887 B | 887 B | ✓ |
| index-08fb3f..c0e9.js gzip | 227 B | 227 B | ✓ |
| link-6f8445b..99e1.js gzip | 1.3 kB | 1.3 kB | ✓ |
| routerDirect..8aa1.js gzip | 284 B | 284 B | ✓ |
| withRouter-f..e777.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.72 kB | 7.72 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| _app-669dbe5..dule.js gzip | 626 B | 626 B | ✓ |
| _error-d5979..dule.js gzip | 2.3 kB | 2.3 kB | ✓ |
| hooks-805c40..dule.js gzip | 387 B | 387 B | ✓ |
| index-6ba5a4..dule.js gzip | 226 B | 226 B | ✓ |
| link-91516ae..dule.js gzip | 1.25 kB | 1.25 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-d..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.36 kB | 5.36 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| _buildManifest.js gzip | 274 B | 274 B | ✓ |
| _buildManife..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 556 B | 556 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| _error.js | 1.02 MB | 1.02 MB | ✓ |
| 404.html | 4.18 kB | 4.18 kB | ✓ |
| hooks.html | 3.82 kB | 3.82 kB | ✓ |
| index.js | 1.02 MB | 1.02 MB | ✓ |
| link.js | 1.06 MB | 1.06 MB | ✓ |
| routerDirect.js | 1.05 MB | 1.05 MB | ✓ |
| withRouter.js | 1.05 MB | 1.05 MB | ✓ |
| Overall change | 5.2 MB | 5.2 MB | ✓ |

</details>

</details>
Commit: 29d54f6d3c53a95f0105f08f031a8583511f4112

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| buildDuration | 14.2s | 14.4s | ⚠️ +148ms |
| nodeModulesSize | 65.5 MB | 65.5 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.594 | 2.669 | ⚠️ +0.08 |
| / avg req/sec | 963.76 | 936.65 | ⚠️ -27.11 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.702 | 1.534 | -0.17 |
| /error-in-render avg req/sec | 1468.67 | 1629.57 | +160.9 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| 677f882d2ed8..b7a9.js gzip | 10.2 kB | 10.2 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-935e413..31c4.js gzip | 6.76 kB | 6.76 kB | ✓ |
| webpack-488d..c0e7.js gzip | 751 B | 751 B | ✓ |
| Overall change | 56.8 kB | 56.8 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 6.09 kB | 6.09 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-4b8f4a8..dule.js gzip | 5.84 kB | 5.84 kB | ✓ |
| webpack-4f62..dule.js gzip | 751 B | 751 B | ✓ |
| Overall change | 51.8 kB | 51.8 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| polyfills-05..1236.js gzip | 30.8 kB | 30.8 kB | ✓ |
| Overall change | 30.8 kB | 30.8 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| _app-8f5f611..1f7b.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-a98d9..5cb7.js gzip | 3.45 kB | 3.45 kB | ✓ |
| hooks-f7f3d0..7465.js gzip | 887 B | 887 B | ✓ |
| index-08fb3f..c0e9.js gzip | 227 B | 227 B | ✓ |
| link-6f8445b..99e1.js gzip | 1.3 kB | 1.3 kB | ✓ |
| routerDirect..8aa1.js gzip | 284 B | 284 B | ✓ |
| withRouter-f..e777.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.72 kB | 7.72 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| _app-669dbe5..dule.js gzip | 626 B | 626 B | ✓ |
| _error-d5979..dule.js gzip | 2.3 kB | 2.3 kB | ✓ |
| hooks-805c40..dule.js gzip | 387 B | 387 B | ✓ |
| index-6ba5a4..dule.js gzip | 226 B | 226 B | ✓ |
| link-91516ae..dule.js gzip | 1.25 kB | 1.25 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-d..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.36 kB | 5.36 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| _buildManifest.js gzip | 274 B | 274 B | ✓ |
| _buildManife..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 556 B | 556 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| index.html gzip | 945 B | 945 B | ✓ |
| link.html gzip | 953 B | 953 B | ✓ |
| withRouter.html gzip | 940 B | 940 B | ✓ |
| Overall change | 2.84 kB | 2.84 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| buildDuration | 16s | 16.1s | ⚠️ +15ms |
| nodeModulesSize | 65.5 MB | 65.5 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| 677f882d2ed8..b7a9.js gzip | 10.2 kB | 10.2 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-935e413..31c4.js gzip | 6.76 kB | 6.76 kB | ✓ |
| webpack-488d..c0e7.js gzip | 751 B | 751 B | ✓ |
| Overall change | 56.8 kB | 56.8 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 6.09 kB | 6.09 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-4b8f4a8..dule.js gzip | 5.84 kB | 5.84 kB | ✓ |
| webpack-4f62..dule.js gzip | 751 B | 751 B | ✓ |
| Overall change | 51.8 kB | 51.8 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| polyfills-05..1236.js gzip | 30.8 kB | 30.8 kB | ✓ |
| Overall change | 30.8 kB | 30.8 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| _app-8f5f611..1f7b.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-a98d9..5cb7.js gzip | 3.45 kB | 3.45 kB | ✓ |
| hooks-f7f3d0..7465.js gzip | 887 B | 887 B | ✓ |
| index-08fb3f..c0e9.js gzip | 227 B | 227 B | ✓ |
| link-6f8445b..99e1.js gzip | 1.3 kB | 1.3 kB | ✓ |
| routerDirect..8aa1.js gzip | 284 B | 284 B | ✓ |
| withRouter-f..e777.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.72 kB | 7.72 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| _app-669dbe5..dule.js gzip | 626 B | 626 B | ✓ |
| _error-d5979..dule.js gzip | 2.3 kB | 2.3 kB | ✓ |
| hooks-805c40..dule.js gzip | 387 B | 387 B | ✓ |
| index-6ba5a4..dule.js gzip | 226 B | 226 B | ✓ |
| link-91516ae..dule.js gzip | 1.25 kB | 1.25 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-d..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.36 kB | 5.36 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| _buildManifest.js gzip | 274 B | 274 B | ✓ |
| _buildManife..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 556 B | 556 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| _error.js | 1.02 MB | 1.02 MB | ✓ |
| 404.html | 4.18 kB | 4.18 kB | ✓ |
| hooks.html | 3.82 kB | 3.82 kB | ✓ |
| index.js | 1.02 MB | 1.02 MB | ✓ |
| link.js | 1.06 MB | 1.06 MB | ✓ |
| routerDirect.js | 1.05 MB | 1.05 MB | ✓ |
| withRouter.js | 1.05 MB | 1.05 MB | ✓ |
| Overall change | 5.2 MB | 5.2 MB | ✓ |

</details>

</details>
Commit: fe1d0952ec5d4f83cb3f3bc5d8ce4132d52b43e3

File: examples/with-react-md-typescript/README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+Install it and run:
+
+```bash
+npm install
+npm run dev
+# or
+yarn
+yarn dev
+```
+
```
Comment (Timer) [inline examples/with-react-md-typescript/README.md:57]:
```suggestion
```

File: examples/with-react-md-typescript/pages/_app.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+export default function App({ Component, pageProps }: AppProps): ReactElement {
+  return (
+    <Layout>
+      <Head>
+        <link
+          key="Roboto"
+          rel="stylesheet"
+          href="https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap"
+        />
```
Comment (Timer) [inline examples/with-react-md-typescript/pages/_app.tsx:17]: I'd prefer not to ship a custom font to an example. These are common sources of performance problems.
```suggestion
```

File: examples/with-react-md/pages/_app.jsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+export default function App({ Component, pageProps }) {
+  return (
+    <Layout>
+      <Head>
+        <link
+          key="Roboto"
+          rel="stylesheet"
+          href="https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap"
+        />
```
Comment (Timer) [inline examples/with-react-md/pages/_app.jsx:16]:
```suggestion
```

File: examples/with-react-md/README.md [hunk 4]
```diff
@@ -1,14 +1,27 @@
...
+# ReactMD and Next.js Example
+
+This example sets up a simple [ReactMD](https://github.com/mlaursen/react-md) and Next.js
+app featuring:
+
+- [\_variables.scss](./styles/_variables.scss) to override the default
+  `react-md` theme and feature toggles
+- [app.scss](./styles/app.scss) global styles that conditionally apply the dark theme
+  based on the user's OS preferences
+- a custom [\_app.jsx](./pages/_app.jsx) that imports the Roboto font from
```
Comment (Timer) [inline examples/with-react-md/README.md:10]:
```suggestion
```

File: examples/with-react-md-typescript/README.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+This example sets up a simple [ReactMD](https://github.com/mlaursen/react-md), Next.js, and Typescript
+app featuring:
+
+- [\_variables.scss](./styles/_variables.scss) to override the default
+  `react-md` theme and feature toggles
+- [app.scss](./styles/app.scss) global styles that conditionally apply the dark theme
+  based on the user's OS preferences
+- a custom [\_app.tsx](./pages/_app.tsx) that imports the Roboto font from
+  [Google Fonts](https://fonts.google.com) and uses a persistent layout
```
Comment (Timer) [inline examples/with-react-md-typescript/README.md:11]:
```suggestion
- a custom [\_app.tsx](./pages/_app.tsx) that uses a persistent layout
```

**Review by Timer — CHANGES_REQUESTED** [review]

File: examples/with-react-md-typescript/pages/_app.tsx (hunk 2 above)
Comment (mlaursen) [reply examples/with-react-md-typescript/pages/_app.tsx]: Sounds good, should be removed now!

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| buildDuration | 12.3s | 12.5s | ⚠️ +169ms |
| nodeModulesSize | 66 MB | 66 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.191 | 2.232 | ⚠️ +0.04 |
| / avg req/sec | 1141.22 | 1120 | ⚠️ -21.22 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.199 | 1.198 | 0 |
| /error-in-render avg req/sec | 2084.87 | 2086.39 | +1.52 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| 677f882d2ed8..afbf.js gzip | 10.2 kB | 10.2 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-76d73eb..95d8.js gzip | 6.73 kB | 6.73 kB | ✓ |
| webpack-ccf5..276a.js gzip | 751 B | 751 B | ✓ |
| Overall change | 56.8 kB | 56.8 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 6.11 kB | 6.11 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-0b6ca7d..dule.js gzip | 5.81 kB | 5.81 kB | ✓ |
| webpack-10c7..dule.js gzip | 751 B | 751 B | ✓ |
| Overall change | 51.8 kB | 51.8 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| polyfills-75..1629.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| _app-874bd8a..0103.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-fa39c..ec40.js gzip | 3.45 kB | 3.45 kB | ✓ |
| hooks-585f07..95a3.js gzip | 887 B | 887 B | ✓ |
| index-c7b63f..fc02.js gzip | 227 B | 227 B | ✓ |
| link-e97733c..af21.js gzip | 1.3 kB | 1.3 kB | ✓ |
| routerDirect..ebc7.js gzip | 284 B | 284 B | ✓ |
| withRouter-2..db68.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.72 kB | 7.72 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| _app-97e743e..dule.js gzip | 626 B | 626 B | ✓ |
| _error-b4004..dule.js gzip | 2.3 kB | 2.3 kB | ✓ |
| hooks-696209..dule.js gzip | 387 B | 387 B | ✓ |
| index-a4dd74..dule.js gzip | 226 B | 226 B | ✓ |
| link-fbb1d2e..dule.js gzip | 1.25 kB | 1.25 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-1..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.36 kB | 5.36 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| _buildManifest.js gzip | 272 B | 272 B | ✓ |
| _buildManife..dule.js gzip | 280 B | 280 B | ✓ |
| Overall change | 552 B | 552 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| index.html gzip | 945 B | 945 B | ✓ |
| link.html gzip | 953 B | 953 B | ✓ |
| withRouter.html gzip | 939 B | 939 B | ✓ |
| Overall change | 2.84 kB | 2.84 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| buildDuration | 13.8s | 13.7s | -126ms |
| nodeModulesSize | 66 MB | 66 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| 677f882d2ed8..afbf.js gzip | 10.2 kB | 10.2 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-76d73eb..95d8.js gzip | 6.73 kB | 6.73 kB | ✓ |
| webpack-ccf5..276a.js gzip | 751 B | 751 B | ✓ |
| Overall change | 56.8 kB | 56.8 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 6.11 kB | 6.11 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| main-0b6ca7d..dule.js gzip | 5.81 kB | 5.81 kB | ✓ |
| webpack-10c7..dule.js gzip | 751 B | 751 B | ✓ |
| Overall change | 51.8 kB | 51.8 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| polyfills-75..1629.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| _app-874bd8a..0103.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-fa39c..ec40.js gzip | 3.45 kB | 3.45 kB | ✓ |
| hooks-585f07..95a3.js gzip | 887 B | 887 B | ✓ |
| index-c7b63f..fc02.js gzip | 227 B | 227 B | ✓ |
| link-e97733c..af21.js gzip | 1.3 kB | 1.3 kB | ✓ |
| routerDirect..ebc7.js gzip | 284 B | 284 B | ✓ |
| withRouter-2..db68.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.72 kB | 7.72 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| _app-97e743e..dule.js gzip | 626 B | 626 B | ✓ |
| _error-b4004..dule.js gzip | 2.3 kB | 2.3 kB | ✓ |
| hooks-696209..dule.js gzip | 387 B | 387 B | ✓ |
| index-a4dd74..dule.js gzip | 226 B | 226 B | ✓ |
| link-fbb1d2e..dule.js gzip | 1.25 kB | 1.25 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-1..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.36 kB | 5.36 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| _buildManifest.js gzip | 272 B | 272 B | ✓ |
| _buildManife..dule.js gzip | 280 B | 280 B | ✓ |
| Overall change | 552 B | 552 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | mlaursen/next.js update-react-md-examples | Change |
| - | - | - | - |
| _error.js | 1.02 MB | 1.02 MB | ✓ |
| 404.html | 4.18 kB | 4.18 kB | ✓ |
| hooks.html | 3.82 kB | 3.82 kB | ✓ |
| index.js | 1.02 MB | 1.02 MB | ✓ |
| link.js | 1.06 MB | 1.06 MB | ✓ |
| routerDirect.js | 1.05 MB | 1.05 MB | ✓ |
| withRouter.js | 1.05 MB | 1.05 MB | ✓ |
| Overall change | 5.2 MB | 5.2 MB | ✓ |

</details>

</details>
Commit: f4bba617854b44eda3f45c508a10045a9d7a37c5

**Review by Timer — APPROVED** [review]
LGTM, thanks!

**Pull request merged**